File: app/src/main/java/bj/rxjavaexperimentation/model/collectionrelease/RootCollectionRelease.java

package bj.rxjavaexperimentation.model.collectionrelease;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import bj.rxjavaexperimentation.model.common.Pagination;

public class RootCollectionRelease
{

    @SerializedName("pagination")
    @Expose
    private Pagination pagination;
    @SerializedName("releases")
    @Expose
    private List<CollectionRelease> collectionReleases = null;

    public Pagination getPagination() {
        return pagination;
    }

    public void setPagination(Pagination pagination) {
        this.pagination = pagination;
    }

    public List<CollectionRelease> getCollectionReleases() {
        return collectionReleases;
    }

    public void setCollectionReleases(List<CollectionRelease> collectionReleases) {
        this.collectionReleases = collectionReleases;
    }

}

File: app/src/main/java/bj/vinylbrowser/model/collection/RootCollectionRelease.kt
package bj.vinylbrowser.model.collection

import bj.vinylbrowser.model.common.Pagination
import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class RootCollectionRelease(val pagination: Pagination,
                                 @SerializedName("releases") val collectionReleases: List<CollectionRelease>)